src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Policy/UnconditionalPolicy.cs

```diff
@@ -15,7 +15,7 @@ internal interface IIdentityInfo
         IIdentity Identity { get; }
     }
 
-    internal class UnconditionalPolicy : IAuthorizationPolicy, IDisposable
+    public class UnconditionalPolicy : IAuthorizationPolicy, IDisposable
     {
```

Comment:
Why are you making this public? It's internal on .NET Framework.

Reply:
Same as above, the accessor changes to this file are from a previous implementation which I eventually scrapped. Will revert.